README.md

```diff
@@ -1 +1,147 @@
-# Tech-Forum
\ No newline at end of file
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+# 📖 [Tech Forum Page] <a name="about-project"></a>
+
+**[Tech Forum Page]]** is a capstone based project that is aimed enabling individual development of web pages. It is a personalized online website that allows me to have something unique in my portfolio to share with potential employers during job searching.
+
+## 🛠 Built With 
+<a name="built-with">-CSS</a>
+<a name="built-with">-HTML</a>
+<a name="built-with">-JavaScript</a>
+
+### Key Features <a name="key-features"></a>
+
+- **[A home page]**
+- **[An about page]**
+- **[Mobile and desktop view due responsiveness]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://yourdeployedapplicationlink.com)
+
```

Comment:
- [ ] Kindly update the correct link to your live demo, I noticed you had deployed your project using GitHub pages as expected. 👍

Reply:
- [ ] @chepkok3 as previously requested kindly add the correct link to your project demo.